plaid/accounts.go

```diff
@@ -30,10 +30,15 @@ type AccountNumber struct {
 	WireRouting string `json:"wire_routing"`
 }
 
+type getBalancesRequestOptions struct {
+	AccountIDs []string `json:"account_ids,omitempty"`
+}
+
 type getBalancesRequest struct {
-	ClientID    string `json:"client_id"`
-	Secret      string `json:"secret"`
-	AccessToken string `json:"access_token"`
+	ClientID    string                    `json:"client_id"`
+	Secret      string                    `json:"secret"`
+	AccessToken string                    `json:"access_token"`
+	Options     getBalancesRequestOptions `json:"options,omitempty"`
```

Comment:
interestingly, `omitempty` will not cause the field to be omitted when no options are supplied, since empty structs do not provoke that behavior.

https://www.sohamkamani.com/blog/golang/2018-07-19-golang-omitempty/

I think for our purposes, this is fine.

Reply:
This is really interesting. Was following https://github.com/plaid/plaid-go/blob/1109794cef7b985dd9519a6d4439fd4bd4cdd2ea/plaid/accounts.go#L57

Thanks for the link though, it was pretty informative :)